tests/functional/mailcheck.js

```diff
@@ -29,27 +26,17 @@ define([
       return this.remote.then(clearBrowserState());
     },
 
-    'treatment works': function () {
+    'tooltip works': function () {
       var BAD_EMAIL = 'something@gnail.com';
       var CORRECTED_EMAIL = 'something@gmail.com';
 
       return this.remote
-        .then(openPage(EXP_MAILCHECK_URL + EXP_TREATMENT, '#fxa-signup-header'))
+        .then(openPage(EXP_MAILCHECK_URL, '#fxa-signup-header'))
         .then(type('.email', BAD_EMAIL))
-        .then(click('.password'))
+        .then(click('#age'))
```

Comment:
@divyabiyani can you check if adding a `.sleep(1000)` helps this, maybe there is an issue with tooltip fade on CI, example:

```
       .then(openPage(EXP_MAILCHECK_URL, '#fxa-signup-header'))
        .then(type('.email', BAD_EMAIL))
        .then(click('#age'))
        // give time for the tooltip to fade in
        .sleep(1000)
        .then(click('.tooltip-suggest > span:nth-child(1)'))
```